suites/quincy/baremetal_pipeline/tier-1_rgw_ms.yaml

```diff
@@ -448,3 +448,49 @@ tests:
             module: push_cosbench_workload.py
             name: push cosbench push workload from secondary
             polarion-id: CEPH-83575073
+
+  - test:
+      name: Parallel delete of objects from bucket
+      desc: Parallel delete of objects from bucket
+      module: test_parallel.py
+      parallel:
+        - test:
+            abort-on-fail: true
+            clusters:
+              ceph-pri:
+                config:
+                  controllers:
+                    - folio01
+                  drivers:
+                    - folio01
+                    - folio02
+                    - folio03
+                  fill_percent: 30
+                  workload_type: cleanup
+                  record_sync_on_site: ceph-sec
+                  bucket_prefix: cosbench01-bkt-
+                  number_of_buckets: 1
+            desc: initiate cosbench cleanup workload from primary
+            module: push_cosbench_workload.py
+            name: push cosbench cleanup workload from primary
+            polarion-id: CEPH-83575074
+        - test:
+            abort-on-fail: true
+            clusters:
+              ceph-sec:
+                config:
+                  controllers:
+                    - folio04
+                  drivers:
+                    - folio04
+                    - folio05
+                    - folio06
+                  fill_percent: 30
```

Comment:
@anrao19 , I think we need to add support for number of objects and object range along with fill_percent
because fill percent might work for fill workload, but according to the remaining space available the objects_count differ for cleanup workload. exact objects will not get deleted with cleanup
we can take it as an enhancement in the coming PR's

Reply:
Sure @hmaheswa

Reply:
done changes